transactions/src/main/java/org/jboss/as/txn/service/JTAEnvironmentBeanService.java

```diff
@@ -56,7 +57,9 @@ public void start(StartContext context) throws StartException {
         // recovery nodes
         jtaEnvironmentBean.setXaRecoveryNodes(Collections.singletonList(nodeIdentifier));
         // setup the XA orphan filters
-        jtaEnvironmentBean.setXaResourceOrphanFilterClassNames(Arrays.asList(JTATransactionLogXAResourceOrphanFilter.class.getName(), JTANodeNameXAResourceOrphanFilter.class.getName(), SubordinateJTAXAResourceOrphanFilter.class.getName()));
+        if (System.getProperty("JTAEnvironmentBean.xaResourceOrphanFilterClassNames") == null) {
```

Comment:
I think a DMR operation might be a different way to configure this list but for now I think we should go with the option that allows us to interpret what the user is requesting which may be an empty list (NOT a good idea but neither would be a partial list)